test/util/git.ts

```diff
@@ -0,0 +1,38 @@
+import { SpawnOptions } from "child_process";
+import * as fs from "fs-extra";
+import "mocha";
+import * as path from "path";
+import * as assert from "power-assert";
+import { obtainGitInfo } from "../../src/internal/env/gitInfo";
+
```

Comment:
Where did this come from? Why is the source and the test in this file under test?

Reply:
I needed a way to create the git-info.json but can’t rely on cli ad that depends on client. I wanted something that doesn’t end up in our source tree.